sdcm/utils/monitorstack.py

```diff
@@ -249,3 +277,68 @@ def is_docker_available():
     else:
         LOGGER.warning('Docker is not available on your computer. Please install docker software before continue')
         return False
+
+
+def verify_monitor_stack(monitor_stack_dir, scylla_version):
```

Comment:
shouldn't all the verify function would be with a retrying decorator ?
i.e. couldn't be that we would test before they are ready ?

Reply:
No. The testing could start only after docker will be started.